src/pages/AddItem.js

```diff
@@ -1,13 +1,27 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { withFirestore } from 'react-firestore';
 import AddItemHeader from './AddItemHeader';
+import getToken from '../lib/token';
+import NavTabs from '../components/NavTabs';
 
 const AddItem = ({ firestore }) => {
   const [name, setName] = useState('');
 
+  const initialToken = () => window.localStorage.getItem('token') || getToken();
+  const [token] = useState(initialToken);
+  // const[ redirect, setRedirect ] = useState(false)
+  useEffect(() => {
+    window.localStorage.setItem('token', token);
+  }, [token]);
+
+  console.log('token:', token);
```

Comment:
Before merging, please remove `console.log` statements and commented out code. Thanks!

Reply:
I think I got them all https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/49/commits/1039ba7f558d56e10488afb203aeecf8357f302a